bin/update-ips.bash

```diff
@@ -235,12 +235,30 @@ fi
 
 ## Add sc nodes ips to sc config
 if [[ "${CHECK_CLUSTER}" =~ ^(sc|both)$ ]]; then
-    checkIfDiffAndUpdateKubectlIPs "sc" "" ".networkPolicies.global.scNodes.ips" "${config["override_sc"]}"
+    if [[ "${CK8S_CLOUD_PROVIDER}" =~ ^(citycloud|elastx)$ ]]; then
+        echo "Since your provider is either citycloud or elastx, you can choose to add the internal network(172.16.0.0/32) instead of specific IPs."
+        echo -n "- run? [y/N]: "
+        read -r reply
+        if [[ "${reply}" != "N" ]]; then
+            checkIfDiffAndUpdateKubectlIPs "sc" "" "172.16.0.0/32" "${config["override_sc"]}"
```

Comment:
You need to check which IPs the nodes have and infer the subnet, since it is not a given that they all use `172.16.0.0`. Also to get the entire subnet it should be `/24` for `172.16.0.x` else it will only be the specific IP ending in 0.

Additionally, it would be nice if it checked that node IPs fall into the set subnet CIDR, and won't prompt if that is the case.

Reply:
https://www.arin.net/reference/research/statistics/address_filters/ .. Apparently 172.16.0.0/12 could do the trick as far as coverage is concerned.